packages/rum/src/domain/record/serializationUtils.ts

```diff
@@ -69,3 +70,38 @@ export function getElementInputValue(element: Element, nodePrivacyLevel: NodePri
 
   return value
 }
+
+export const URL_IN_CSS_REF = /url\((?:(')([^']*)'|(")([^"]*)"|([^)]*))\)/gm
+export const ABSOLUTE_URL = /^[A-Za-z]+:|^\/\//
+export const DATA_URI = /^data:.*,/i
+
+export function switchToAbsoluteUrl(cssText: string, cssHref: string | null): string {
+  return cssText.replace(
+    URL_IN_CSS_REF,
+    (
+      matched: string,
+      singleQuote: string | undefined,
+      firstPathMatched: string | undefined,
+      doubleQuote: string | undefined,
+      secondPathMatched: string | undefined,
+      thirdPathMatched: string | undefined
+    ) => {
+      const path = firstPathMatched || secondPathMatched || thirdPathMatched
```

Comment:
Thanks for the explanation. Yet, I disagree about renaming `matched` to `urlFunction`. I renamed it to `matchingSubstring` since it is a substring that matches the regex